src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/AssessmentTransformer.kt

```diff
@@ -111,20 +111,23 @@ class AssessmentTransformer(
   }
 
   fun getSortedReferralHistoryNotes(
-    jpa: TemporaryAccommodationAssessmentEntity,
+    assessment: TemporaryAccommodationAssessmentEntity,
     cas3Events: List<DomainEventEntity>,
+    filterUserNotes: Boolean = false,
   ): List<ReferralHistoryNote> {
     val lastReferralRejectedHistoryNote =
-      jpa.referralHistoryNotes.filter { it is AssessmentReferralHistorySystemNoteEntity && it.type == ReferralHistorySystemNoteType.REJECTED }
+      assessment.referralHistoryNotes.filter { it is AssessmentReferralHistorySystemNoteEntity && it.type == ReferralHistorySystemNoteType.REJECTED }
         .maxByOrNull { it.createdAt }
 
-    val notes = jpa.referralHistoryNotes.map {
+    val notes = assessment.referralHistoryNotes.map {
       if (it.id == lastReferralRejectedHistoryNote?.id) {
-        assessmentReferralHistoryNoteTransformer.transformJpaToApi(it, jpa)
+        assessmentReferralHistoryNoteTransformer.transformJpaToApi(it, assessment)
       } else {
         assessmentReferralHistoryNoteTransformer.transformJpaToApi(it)
       }
-    }.toMutableList()
+    }
+      .filter { it.type != "user" || !filterUserNotes }
```

Comment:
it would be nice if this was an enumerated type - if that's going to cause UI issues could we maybe have constants defined in assessmentReferralHistoryNoteTransformer for the different possible values and refer to that here instead?